torchtune/modules/transforms/vision_utils/tile_crop.py

```diff
@@ -11,49 +11,53 @@
 logger = logging.getLogger(__name__)
```

Comment:
not part of your refactor, but i dont think we are using it. Do you mind deleting?